common/cpp/src/google_smart_card_common/numeric_conversions.h

```diff
@@ -96,12 +93,13 @@ inline bool CastInt64ToInteger(
     *result = static_cast<T>(value);
     return true;
   }
-  *error_message = FormatBoostFormatTemplate(
-      internal::kErrorNumberOutsideTypeRange,
-      type_name,
+  *error_message = FormatPrintfTemplate(
+      "The integer value is outside the range of type \"%s\": %" PRId64 " not "
+      "in [%" PRId64 "; %" PRIu64 "] range",
+      type_name.c_str(),
       value,
-      std::numeric_limits<T>::min(),
-      std::numeric_limits<T>::max());
+      static_cast<int64_t>(std::numeric_limits<T>::min()),
+      static_cast<uint64_t>(std::numeric_limits<T>::max()));
```

Comment:
nit: I think intmax_t and uintmax_t would be better here

Reply:
Done.
Thanks for the tip! Didn't know about this thing.